test/performance/odp_sched_perf.c

```diff
@@ -154,8 +158,15 @@ static void print_usage(void)
 	       "  -b, --burst            Maximum number of events per operation. Default: 100.\n"
 	       "  -t, --type             Queue type. 0: parallel, 1: atomic, 2: ordered. Default: 0.\n"
 	       "  -f, --forward          0: Keep event in the original queue, 1: Forward event to the next queue. Default: 0.\n"
-	       "  -a, --fairness         0: Don't count events per queue, 1: Count and report events relative to average. Default: 0.\n"
+	       "  -F, --fairness         0: Don't count events per queue, 1: Count and report events relative to average. Default: 0.\n"
 	       "  -w, --wait_ns          Number of nsec to wait before enqueueing events. Default: 0.\n"
+	       "  -S, --stress           CPU stress function to be called for each event data word (uint32_t),\n"
+	       "                         multiple flags may be selected:\n"
```

Comment:
It would be good to document that this requires also setting `-m` or/and `-n` options.  Could also throw an error if they are missing.

Reply:
Fixed in v3